Makefile

```diff
@@ -32,6 +32,19 @@ test-before-build:
 	## Check for unnecessarily fully qualified URLs (breaks local previews and internal link checking)
 	! git --no-pager grep -- '^\[.*]: https://bitcoinops.org' | grep -v skip-test | grep .
 	! git --no-pager grep -- '](https://bitcoinops.org' | grep -v skip-test | grep .
+	## Check for duplicate words
+	@ ## Only applies to *.md files
+	@ ## Add <!-- skip-test --> to same line to skip this test
```

Comment:
Perhaps change this to skip-duplicate-words-test or similar in case we add any more linters.

Reply:
@harding do you think this is worth updating?

Reply:
ha. You just did. Thanks!